frontend_web_service/docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+version: '3'
+
+services:
+  auth_service:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    container_name: frontend_web_service
+    ports:
+      - 3000:3000
```

Comment:
@raquevedo esto no nos genera conflicto con el be auth, creo que estaba en el mismo puerto

Reply:
Los contenedores de docker son como IPs diferentes, no se bien como funciona cuando estan en la misma red pero eso recien es un problema cuando lo integramos con Kong